package.json

```diff
@@ -44,12 +44,12 @@
     "release:major": "npm version major && git push && git push --tags && npm publish",
     "docker:build": "scripts/docker/build.sh",
     "docker:build-and-push": "npm run docker:build && scripts/docker/push.sh",
-    "docker:pull": "docker pull augurproject/dev-node-parity:latest && docker pull augurproject/dev-node-geth:latest && docker pull augurproject/dev-pop-geth:core-$(npm view augur-core version) && docker pull augurproject/dev-pop-normtime-geth:latest",
+    "docker:pull": "docker pull augurproject/dev-node-parity:latest && docker pull augurproject/dev-node-geth:latest && docker pull augurproject/dev-pop-geth:core-$(node scripts/package-version.js augur-core) && docker pull augurproject/dev-pop-normtime-geth:latest",
     "docker:geth": "docker run -e GETH_VERBOSITY=4 -it -p 8545:8545 -p 8546:8546 -v $HOME/docker:/geth/chain augurproject/dev-node-geth:latest",
```

Comment:
augurproject/dev-pop-normtime-geth:latest",
do we want to get rid of ;latest for normtime as well?